MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+---
+layout: post
+title: Working with ListView in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Working with ListView support in Syncfusion .NET MAUI ListView control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Working with ListView in .NET MAUI ListView (SfListView)
+
+## Interacting with ListView Items
+
+### Loaded event
+
+The [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event is raised when the `SfListView` is loading in view for the first time.
+
+{% tabs %}
+{% highlight c# %}
+
+listView.Loaded += ListView_Loaded;
+
+private void ListView_Loaded(object sender, Syncfusion.Maui.ListView.ListViewLoadedEventArgs e)
+{
+   listView.SelectedItems.Add(viewModel.Customers[2]);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The `Loaded` event used for the following use cases:
+
+* To scroll the desired position or index by using the [ScrollTo](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ScrollTo_System_Double_System_Boolean_) or [ScrollToRowIndex](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ListViewLayout.html#Syncfusion_Maui_ListView_ListViewLayout_ScrollToRowIndex_System_Int32_Microsoft_Maui_Controls_ScrollToPosition_System_Boolean_).
+* To collapse all the groups.
+* To find the sorted or grouped [DataSource.DisplayItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_DisplayItems) of underlying bound data to `SfListView`.
+
+### Tapped event
+
+The [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event will be triggered whenever tapping the item. Here, [TapCommandParameter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommandParameter) property sets the parameter for [SfListView.TapCommand](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommand) and its default value is [ItemTappedEventArgs](). `ItemTappedEventArgs` has the following members which provides the information for `ItemTapped` event:

```

Comment:
The [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event will be triggered whenever tapping the item. Here, [TapCommandParameter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommandParameter) property sets the parameter for [SfListView.TapCommand](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommand) and its default value is [ItemTappedEventArgs](). `ItemTappedEventArgs` has the following members, which provide information for the `ItemTapped` event:

Reply:
I have modified the line as per the concerns provided